app/Http/Controllers/Front/BaseClientController.php

```diff
@@ -58,12 +56,7 @@ public function getExpiryDate($orders)
         $end = '--';
         if ($orders->subscription()->first()) {
             if (strtotime($orders->subscription()->first()->update_ends_at) > 1) {
-                $ends = new DateTime($orders->subscription()->first()->update_ends_at);
-                $tz = \Auth::user()->timezone()->first()->name;
-                $ends->setTimezone(new DateTimeZone($tz));
-                $date = $ends->format('M j, Y, g:i a ');
-                $end = $date;
-                // dd($end);
+                $end = getExpiryLabel($orders->subscription()->first()->update_ends_at,'badge');
```

Comment:
please do not query the entire object if you need only `update_ends_at`

Reply:
This method is not in use now. Removed it